### ISSUE TITLE
Can this module be integrated with wcag-validator? if yes please guide me how. I want to use it for accessibility testing.

### ISSUE THREAD

**Issue by jatinsi** [tmcw/wcag-contrast#4]
Can this module be integrated with wcag-validator? if yes please guide me how. I want to use it for accessibility testing.
Wcag-validator does not have the contrast module in it so this seems to be the best fit but I have no idea how to build a js project so need guidance.

**Comment by tmcw** [comment]
There are many WCAG validators: can you narrow down which one you're referring to with a link?

**Comment by jatinsi** [comment]
thanks for your reply, here is the link.

https://github.com/zrrrzzt/wcag-validator

**Comment by tmcw** [comment]
That module hits a web service called Achecker, so it doesn't have any of the accessibility-checking logic in its codebase. So there's not really a route to integrating wcag-contrast with it, sorry.
